main.go

```diff
@@ -28,6 +29,43 @@ func getTokens() *Tokens {
 		"}",
 		":",
 		"\"",
+		",",
```

Comment:
Clean up main.go and get rid of the old code which may cause confusion

Reply:
Keeping the structure for when I have to create the REPL for reading in JSON files to parse